### DISCUSSION TITLE
把特性区域的true改为false之后，没有任何变化，界面还是显示该区域的存在（starter主题）

### DISCUSSION THREAD

**Discussion by KaterinaJiang** [tangly1024/NotionNext#2993] [Q&A]
![image](https://github.com/user-attachments/assets/3e035c20-e91d-40b5-bee5-20ecdf3402f8)

**Comment by LooseLi** [comment]
你好，建议先将博客升级到最新版本，然后在 Notion 的 config 文档中添加 `STARTER_FEATURE_ENABLE` 字段，并将其值设为 `false`。启用该字段后，刷新页面即可。配置流程可参考[NotionNext帮助文档](https://docs.tangly1024.com/article/how-to-config-notion-next)
